<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorMiscWidget</class>
 <widget class="QWidget" name="EditorMiscWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkReadonlySystemHeaders">
     <property name="text">
      <string>Open system header files in read only mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLoadLastFiles">
     <property name="text">
      <string>Auto load files being open when Red Panda C++ last exited.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkUseUTF8ByDefault">
     <property name="text">
      <string>Use UTF-8 as the default encoding for new file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Default file type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbCppFile">
        <property name="text">
         <string>C++ files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCFile">
        <property name="text">
         <string>C files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>117</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
